### ISSUE TITLE
Releasing 0.5.0-beta.2

### ISSUE THREAD

**Issue by krskibin** [pagedjs/pagedjs#213]
Hey, are there any updates on version 0.5.0-beta.1? We are having issues with table splitting and need it fixed as soon as possible.

**Comment by Malcolmnixon** [comment]
Indeed - betas are cheap; and it would be valuable to get more testing before RC or production releases are made.

**Comment by Fxlr8** [comment]
Yes, table wrapping problem is really painful. Need any help with that?

**Comment by RobMayer** [comment]
I haven't gotten to the point where I'm hitting issues specifically with table splitting, but Nigel's recent updates in #196 seemed to have fixed a lot of PagedJS falling over silently during pagination. You might want to fork pagedjs, pull his branch, and do a local build if you're in a position where you don't need the CDN or anything.

**Comment by lunarcloud** [comment]
https://github.com/pagedjs/pagedjs/pull/196#issuecomment-2338706435

**Comment by julientaq** [comment]
thank you @lunarcloud 

@Fxlr8 workin with grid can be dangerous. but i’m curious to see what if fixes and how (grid tends to break when working on page break). If you’d like to share the code, i’d love to look at it
